DATA-5302: improve docs (#132)
adobe/commerce-events: examples/events/readme.md

| CTR              | [`page-view`](./page-view.md),[`recs-request-sent`](./recs-request-sent.md), [`recs-response-received`](./recs-response-received.md), [`recs-unit-render`](./recs-unit-render.md), [`recs-item-click`](./recs-item-click.md), [`recs-add-to-cart-click`](./recs-item-add-to-cart.md)                                          | unitId, sku |
| vCTR             | [`page-view`](./page-view.md),[`recs-request-sent`](./recs-request-sent.md), [`recs-response-received`](./recs-response-received.md), [`recs-unit-render`](./recs-unit-render.md), [`recs-unit-view`](./recs-unit-view.md), [`recs-item-click`](./recs-item-click.md), [`recs-add-to-cart-click`](./recs-item-add-to-cart.md) | unitId, sku |

#### Other use cases and applications

> A number of Adobe Commerce products utilizing commerce event collection send data additional data that is used for Product Analytics and other internal tracking.

Reviewer: Is "Product Analytics" an official product? If not, it should be lowercase.

##### Tracking storefront templates

> Set [strorefront context](./example-contexts/mock-storefront-context.md) to the [storefront template](../../packages/storefront-events-sdk/src/types/schemas/storefrontInstance.ts) for your store. If your storefront is not yet defined in the Commerce Events schema, you can request for it to be added following the contribution guildelines [here](../../CONTRIBUTING.md).

Reviewer: ```suggestion
Set [strorefront context](./example-contexts/mock-storefront-context.md) to the [storefront template](../../packages/storefront-events-sdk/src/types/schemas/storefrontInstance.ts) for your store. If your storefront is not yet defined in the Commerce Events schema, follow the contribution guildelines [here](../../CONTRIBUTING.md) to have it added.
```

And we can remove 2 of the blank lines below.
